cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/DefaultPluginConfigurer.java

```diff
@@ -43,10 +43,10 @@
  */
 public class DefaultPluginConfigurer extends DefaultDatasetConfigurer implements PluginConfigurer {
 
-  private final Id.Artifact artifactId;
-  private final ArtifactRepository artifactRepository;
-  private final PluginInstantiator pluginInstantiator;
-  private final Map<String, Plugin> plugins;
+  protected final Id.Artifact artifactId;
```

Comment:
Just that it seems like arbitrary that it get exposed in the `DefaultPluginConfigurer`.